edx_django_utils/__init__.py

```diff
@@ -2,7 +2,7 @@
 EdX utilities for Django Application development..
 """
 
-__version__ = "3.9.0"
+__version__ = "3.9.1"
```

Comment:
This is adding features, so it should get a different minor version.
```suggestion
__version__ = "3.10.0"
```